tools/lambda-promtail/lambda-promtail/main_test.go

```diff
@@ -1,9 +1,10 @@
 package main
 
 import (
+	"testing"
+
```

Comment:
can we avoid whitespace/import reordering changes, just makes reviews for new features easier :) thank you

Reply:
Sure I will remove them 😅

Reply:
thanks!

whitespace/import cleanup PRs are welcome! I just want to be sure I can easily see everything relevant to the things you're trying to fix here with this PR :)